src/ReactApp.tsx

```diff
@@ -561,10 +561,10 @@ export default function ReactApp(settings: any) {
       let list = generateList(getItemsOfGivenTypeAndStatus(activeTab, SECTIONS[i]));
 
       result.push(
-        <div key={ i } style={{ display: 'flex', flexDirection: 'column' }}>
+        <div key={ i } style={{ display: 'flex', flexDirection: 'column', color: 'var(--text-normal)', paddingLeft: 'clamp(15px, 0.5%, 50px)' }}>
           <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
-              <h3 style={{ marginRight: 5, color: '#ffffff80' }}>{ SECTIONS[i].includes('To') ? SECTIONS[i].replace('To', 'to') : SECTIONS[i] }</h3>
-              { list.length != 0 ? <p onClick={ () => handleSectionHideClick(lowercaseSection) }>{ getTabLedgerSectionHideValue(lowercaseSection) ? 'unhide' : 'hide' }</p> : null }
+              <h3 style={{ marginRight: 5, color: 'var(--h3-color, var(--text-title-h3, var(--text-normal)))', fontSize: 'clamp(16px, 1.25vw, 22px)' }}>{ SECTIONS[i].includes('To') ? SECTIONS[i].replace('To', 'to') : SECTIONS[i] }</h3>
+              { list.length != 0 ? <p onClick={ () => handleSectionHideClick(lowercaseSection) }>{ getTabLedgerSectionHideValue(lowercaseSection) ? 'Show' : 'hide' }</p> : null }
```

Comment:
Can you make 'Show' lowercase? Since 'hide' is lowercase, this would make it consistent.